server/conf/messages.en-US

```diff
@@ -189,6 +189,9 @@ content.notEligible=Not Eligible
 link.edit=Edit
 link.answer=Answer
 
+# The text that appears next to a question that was answered in another program to let the user know the date it was previously answered on.
+content.previouslyAnsweredOn=Previously answered on
```

Comment:
**issue (blocking):** I believe you should include the date substitution here so it can be properly placed in the translated content.  See the other examples with `{0}`

Reply:
I'm not sure what you mean by resubmit, but yeah it's a different message since it includes the substitution.

Reply:
Sorry I meant create a new localization request with the {0}.